olm-catalog/serverless-operator/1.7.0/operator_v1alpha1_knativeeventing_crd.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: knativeeventings.operator.knative.dev
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.version
+    name: Version
+    type: string
+  - JSONPath: .status.conditions[?(@.type=="Ready")].status
+    name: Ready
+    type: string
+  - JSONPath: .status.conditions[?(@.type=="Ready")].reason
+    name: Reason
+    type: string
+  group: operator.knative.dev
+  names:
+    kind: KnativeEventing
+    listKind: KnativeEventingList
+    plural: knativeeventings
+    singular: knativeeventing
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
```

Comment:
Does this conform to what's needed by k8s here? Does `oc explain knativeeventings.operator.knative.dev` return something remotely useful?

Reply:
@markusthoemmes 

```
[aliok@localhost serverless-operator]$ oc explain knativeeventings.operator.knative.dev
KIND:     KnativeEventing
VERSION:  operator.knative.dev/v1alpha1

DESCRIPTION:
     <empty>
```

This is what we get. I think we can improve this in the upstream.

Reply:
Created https://github.com/knative/eventing-operator/issues/162